data/provider/src/main/java/tech/pegasys/teku/api/NodeDataProvider.java

```diff
@@ -13,14 +13,58 @@
 
 package tech.pegasys.teku.api;
 
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import tech.pegasys.teku.datastructures.operations.AttesterSlashing;
+import tech.pegasys.teku.datastructures.operations.ProposerSlashing;
+import tech.pegasys.teku.datastructures.operations.SignedVoluntaryExit;
```

Comment:
my preference would be the internal structures are fully qualified below and the external schema structures are the ones imported. it makes the function prototypes way easier to read, and they're what's being consumed...

Reply:
I did it purely out of the fact that I had more internal structures in the clas than external schema structures. Will change.

Reply:
done.